src/main/java/com/softdevsix/exceptions/ProjectNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.softdevsix.exceptions;
+
+public class ProjectNotFoundException {
```

Comment:
This exception should extend the Runtime exception and have the necessary constructors.

In addition, it should be within a directory based on the layer it belongs to, for example, the domain, only if it is related to the domain or another layer.